docs/content/projects/cat.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "OSCAL CAT"
+description: "OSCAL Catalog Authoring Tool (CAT) - import, edit, and export OSCAL Catalogs from the browser"
+heading: "CAT: OSCAL Catalog Authoring Tool"
+toc:
+  enabled: true
+usabanner: true
+---
+
+## What is CAT?
+
+The OSCAL Catalog Authoring Tool (CAT) enables users to create, scope, tailor, control catalogs in [the OSCAL JSON format](https://pages.nist.gov/OSCAL/reference/latest/catalog/json-reference/).
+
+## How does CAT work, is it right for me?
+
+CAT is designed in a convenient, web-based form factor and user-friendly interface. Users can quickly focus on what matters: creating or customizing their control catalog within the familiar experience of their preferred web browser. The responsive user interface allows users to manipulate controls and a resulting control catalog without complete understanding in OSCAL's information and data models in JSON. The focus is the content.
```

Comment:
Users of the CAT **cannot** "create or customize their control catalog". No change can be saved! No new control can be added.
The description needs to be corrected.

Reply:
There is a UI component with wording that suggested you can just start a profile for a catalog (I know I stand corrected, it cannot generate that) without using the 800-53B baselines. I guess I misinterpreted, and that's ok. I can remove.